### ISSUE TITLE
Fixture files fail with non-standard characters

### ISSUE THREAD

**Issue by paulfalgout** [cypress-io/cypress#168]
```js
...
  .fixture('test').as('fxTest')
  .route(/some\/route/, function() {
    return {
      data: this.fxTest
    };
  })
...
```

where test.json is:
```json
[
  {
    "test": "We’ll"
  }
]
```
The right-single quote character code 8217 makes the route fail.
There are no errors except a red `-` under the routes table for the test.

**Comment by brian-mann** [comment]
This is due to not properly `encodeURI` values set in XHR headers.

**Comment by brian-mann** [comment]
Fixed in [`0.16.3`](https://github.com/cypress-io/cypress/wiki/Changelog#0163-06172016).
